templates/imagehub-gateway.yaml

```diff
@@ -44,11 +44,20 @@ metadata:
   name: imagehub-web
 spec:
   hosts:
-  - "imagehub.meshery.io"
+  - "*"
   gateways:
   - sample-app-gateway
   http:
-  - route:
+  - match:
+    - uri:
+      exact: /
+    - uri:
+      exact: /login
```

Comment:
I don't suppose we need to restrict this on we, match all would work fine @pranav-bhatt